src/spring-cloud/azext_spring_cloud/_validators_enterprise.py

```diff
@@ -17,3 +24,37 @@ def only_support_enterprise(cmd, namespace):
 def not_support_enterprise(cmd, namespace):
     if namespace.resource_group and namespace.service and is_enterprise_tier(cmd, namespace.resource_group, namespace.service):
         raise ClientRequestError("'{}' doesn't support for Enterprise tier Spring instance.".format(namespace.command))
+
+
+def validate_builder_create(cmd, namespace):
+    client = get_client(cmd)
+    try:
+        builder = client.build_service_builder.get(namespace.resource_group,
+                                                   namespace.service,
+                                                   DEFAULT_BUILD_SERVICE_NAME,
+                                                   namespace.name)
+        if builder is not None:
+            raise CLIError('Builder {} already exists.'.format(namespace.name))
```

Comment:
Could you please use a specific error type instead of CLIError?

Reply:
OK, updated.